Src/DependencyCollector/Shared/Implementation/ProfilerSqlCommandProcessing.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System.Data;
+    using System.Data.SqlClient;
+    using Microsoft.ApplicationInsights.DependencyCollector.Implementation.Operation;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    /// <summary>
+    /// Concrete class with all processing logic to generate RDD data from the calls backs
+    /// received from Profiler instrumentation for SQL Command.    
+    /// </summary>
+    internal sealed class ProfilerSqlCommandProcessing : ProfilerSqlProcessingBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ProfilerSqlCommandProcessing"/> class.
+        /// </summary>
+        internal ProfilerSqlCommandProcessing(TelemetryConfiguration configuration, string agentVersion, ObjectInstanceBasedOperationHolder telemetryTupleHolder)
+            : base(configuration, agentVersion, telemetryTupleHolder)
+        {            
+        }
+
+        /// <summary>
+        /// Gets SQL command resource name.
+        /// </summary>
+        /// <param name="thisObj">The SQL command.</param>
+        /// <remarks>Before we have clarity with SQL team around EventSource instrumentation, providing name as a concatenation of parameters.</remarks>
+        /// <returns>The resource name if possible otherwise empty string.</returns>
+        internal override string GetResourceName(object thisObj)
```

Comment:
renamed

---
In reply to: [98011264](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r98011264) [](ancestors = 98011264)